src/i18n/nl_NL.ts

```diff
@@ -196,6 +196,10 @@ export default {
             credits: 5_000,
             coins: 25,
         },
+        {
+            caption: '',
+            color: '',
+        },
```

Comment:
We will have to see wether that works. If not, we will probably need to somehow create a workaround